box-ui/src/components/ts/HomeComponent.tsx

```diff
@@ -15,13 +23,32 @@ const HomeComponent: FunctionComponent = () => {
     useEffect(() => {
         counter > 0 && setTimeout(() => setCounter(counter - 1), 1000);
     }, [counter]);
-
     return (
         <div className='HomeButtonsContainer'>
             <div className='LogoContainer'>
                 <img src={JitsiBoxLogo} alt='logo de la jitsi-box' className='logo' />
             </div>
             <HomeButtonsComponent counter={counter} roomName={information.roomName} domain={information.domain} />
+            <div className='LanguageSupport'>
```

Comment:
I think that it lacks a bit of visibility here, the color is the same as in the background. Usually, language menu is anchored on the top left corner, using flags. Could this be possible to implement ?

Reply:
I chose to put the select on the top bottom that can be seen during the jitsi meet (in case we put it on the top right corner, there will be a superposition with the language select).